test/extended/fixtures/nfs-pv-provider.json

```diff
@@ -0,0 +1,235 @@
+            "protocol": "UDP",
+            "port": 111,
+            "targetPort": 111,
+            "nodePort": 0
+          },
+          {
+            "name": "nfs",
+            "protocol": "TCP",
+            "port": 2049,
+            "targetPort": 2049,
+            "nodePort": 0
+          }
+        ],
+        "selector": {
+          "name": "nfs"
+        },
+        "portalIP": "None",
+        "type": "ClusterIP",
+        "sessionAffinity": "None"
+      },
+      "status": {
+        "loadBalancer": {}
+      }
+    },
+    {
+      "kind": "DeploymentConfig",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "nfs",
+        "creationTimestamp": null
+      },
+      "spec": {
+        "strategy": {
+          "type": "Recreate",
+          "resources": {}
+        },
+        "triggers": [
+          {
+            "type": "ConfigChange"
+          }
+        ],
+        "replicas": 1,
+        "selector": {
+          "name": "nfs"
+        },
+        "template": {
+          "metadata": {
+            "creationTimestamp": null,
+            "labels": {
+              "name": "nfs"
+            }
+          },
+          "spec": {
+            "volumes": [
+              {
+                "name": "nfs-data",
+                "emptyDir": {
+                  "medium": ""
+                }
+              }
+            ],
+            "containers": [
+              {
+                "name": "nfs",
+                "image": "mnagy/nfs-server",
+                "args": [
+                  "/exports/01",
+                  "/exports/02",
+                  "/exports/03",
+                  "/exports/04",
+                  "/exports/05"
+                ],
+                "ports": [
+                  {
+                    "containerPort": 111,
+                    "protocol": "UDP"
+                  },
+                  {
+                    "containerPort": 2049,
+                    "protocol": "TCP"
+                  }
+                ],
+                "env": [
+                ],
+                "volumeMounts": [
+                  {
+                    "name": "nfs-data",
+                    "mountPath": "/exports"
+                  }
+                ],
+                "resources": {},
+                "terminationMessagePath": "/dev/termination-log",
+                "imagePullPolicy": "IfNotPresent",
+                "securityContext": {
+                  "capabilities": {},
+                  "privileged": true
+                }
+              }
+            ],
+            "restartPolicy": "Always",
+            "dnsPolicy": "ClusterFirst"
+          }
+        }
+      },
+      "status": {}
+    },
+    {
+      "apiVersion": "v1",
+      "kind": "PersistentVolume",
+      "metadata": {
+        "name": "nfs01"
+      },
+      "spec": {
+        "capacity": {
+            "storage": "${NFS_PV_CAPACITY}"
+        },
+        "accessModes": [ "ReadWriteOnce" ],
+        "nfs": {
```

Comment:
@mnagy you can try to use a hostPath instead of NFS, it might remove a bit of complexity.
This is what I documented for PostgreSQL https://github.com/openshift/postgresql/tree/master/examples/replica#deployment after talking with @pmorie